### DISCUSSION TITLE
How do I hide the line numbers in, e.g., `consult-outline`?

### DISCUSSION THREAD

**Discussion by NightMachinery** [minad/consult#947] [Q&A] [answered]
How do I hide the line numbers in, e.g., `consult-outline`?

<img width="580" alt="image" src="https://github.com/minad/consult/assets/36224762/0e8211d8-b0fb-4f4e-b538-a35c1a615af3">

**Comment by minad** [comment, marked as answer]
The line number is displayed as an annotation. Try this:

~~~ elisp
(consult-customize consult-outline :annotate nil)
~~~
